### PR TITLE
📝Added new pages. [top.gg]

### PR DESCRIPTION
**Pull request by eren885**
image
image

### CODE REVIEW

File: websites/T/top.gg/presence.ts [hunk 1]
```diff
@@ -7,124 +7,133 @@ presence.on("UpdateData", async () => {
...
+      presenceData.details = "Viewing teams";
+  }
+  else if (window.location.pathname.startsWith("/user/") ||
+      window.location.pathname == "/me") {
+      presenceData.details = "Viewing a profile:";
+      presenceData.state = document.querySelector(".header").textContent;
+  }
+  else if (window.location.pathname.startsWith("/api/docs")) {
+      presenceData.state = "Discord Bot List API Documentation";
+  }
```
Comment (ririxidev) [inline websites/T/top.gg/presence.ts:94]: don't need brackets here

File: websites/T/top.gg/presence.ts [hunk 2]
```diff
@@ -7,124 +7,133 @@ presence.on("UpdateData", async () => {
...
+      }
+      else if (window.location.pathname.startsWith("/servers/new")) {
+          presenceData.details = "Adding a new Discord server...";
+      }
+      else if (window.location.pathname.startsWith("/servers/me")) {
+          presenceData.state = "My servers";
+      }
+      else {
+          presenceData.state = "Discord Servers";
+      }
```
Comment (ririxidev) [inline websites/T/top.gg/presence.ts:130]: don't need brackets here

File: websites/T/top.gg/presence.ts [hunk 3]
```diff
@@ -7,124 +7,133 @@ presence.on("UpdateData", async () => {
...
   else if (document.querySelector("#botlistitle")) {
-    presenceData.state = document.querySelector("#botlistitle").textContent;
+      presenceData.state = document.querySelector("#botlistitle").textContent;
   }
-
-  //If it really finds nothing
   else {
-    presenceData.details = "Viewing something...";
+      presenceData.details = "Viewing something...";
   }
```
Comment (ririxidev) [inline websites/T/top.gg/presence.ts:137]: don't need brackets here

File: websites/T/top.gg/presence.ts [hunk 4]
```diff
@@ -7,124 +7,133 @@ presence.on("UpdateData", async () => {
...
+      else {
+          presenceData.state = "Discord Servers";
+      }
   }
-
-  //If it doesn't fit to anything
   else if (document.querySelector("#botlistitle")) {
-    presenceData.state = document.querySelector("#botlistitle").textContent;
+      presenceData.state = document.querySelector("#botlistitle").textContent;
   }
```
Comment (ririxidev) [inline websites/T/top.gg/presence.ts:134]: don't need brackets here

**Review by ririxidev — CHANGES_REQUESTED** [review]
also im thinking that you could add some sort of privacy mode :/

File: websites/T/top.gg/presence.ts [hunk 5]
```diff
@@ -7,124 +7,133 @@ presence.on("UpdateData", async () => {
...
+      }
+      else if (window.location.pathname == "/moderation/reports") {
+          presenceData.state = "Reports Queue";
+      }
+      else if (window.location.pathname == "/moderation/reviews") {
+          presenceData.state = "Reviews Dashboard";
+      }
+      else if (window.location.pathname.startsWith("/moderation/decline")) {
+          presenceData.state = document.querySelector("#botlistitle").textContent;
+      }
```
[the remark this answers is not in the document]
Comment (eren885) [reply websites/T/top.gg/presence.ts:33]: i didn't code that

**Comment by eren885** [comment]
Imao you know what

**Pull request closed without merging**